CompetitionTemplate/src/competition/subsystems/vision/NetworkedJetsonServer.java

```diff
@@ -97,6 +97,9 @@ public void run() {
 
     @Override
     public boolean isConnectionHealthy() {
-        return Timer.getFPGATimestamp() - lastPacket.getRecieptTimestamp() <= healthyTimeThreshold;
```

Comment:
Thanks for the fix! I think this could be made more concise and easy-to-follow:

```java
if(lastPacket == null) {
    return false;
}

return Timer.getFPGATimestamp() - lastPacket.getRecieptTimestamp() <= healthyTimeThreshold;
```